cognite/neat/rules/importers/_dtdl2rules/_unit_lookup.py

```diff
@@ -0,0 +1,224 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class UnitEntry:
+    semantic_type: str
+    unit_type: str
+    unit: str
+
+
+_UNIT_ENTRIES = [
+    UnitEntry("Acceleration", "AccelerationUnit", "centimetrePerSecondSquared"),
+    UnitEntry("Acceleration", "AccelerationUnit", "gForce"),
```

Comment:
How much is this related to QUDT, which our units in CDF are based on?
It is fine to have as is here, but in general I would be so happy to see that CDF comes pre-builded with controlled vocab of units which are interconnected with different standards:
1. QUDT (as base)
2. DTDL

...